### DISCUSSION TITLE
Keyboard shortcuts for previous / next books

### DISCUSSION THREAD

**Discussion by Lightjohn** [gotson/komga#1665] [Ideas]
In help menu I did not find shortcuts to go to next book.
As Home / End are for beginning / end of book maybe **page up** / **page down** for next / previous book?

**Comment by gotson** [comment]
there's no such shortcut because previous/next book is not something you would do very often. When you get to the last page of a book, you can navigate again and go to next book.

**Reply by Lightjohn** [reply]
My use case is that one: I started re reading a full series and I don't want to watch some albums so sometimes after watching 2 pages I just want to watch next album and this is multiple click with mouse or staying on next page until I reach next album.

Feel free to close if you judge my case too obscur :) 
Thanks for your answer

**Reply by gotson** [reply]
there's a shortcut for first/last page, from there you can easily navigate to the next book
